architecture/customizations.adoc

```diff
@@ -25,27 +25,49 @@ Custom Resource Definitions (CRDs) that are deployed on your cluster.
 |===
 |Configuration resource |Description
 
+|APIServer
+|Configure the API server.
+
 |Authentication
 |You can configure the identity provider for your cluster. See
 xref:../authentication/understanding-identity-provider.adoc#understanding-identity-provider[Understanding identity provider configuration].
 
+|Build
+|Configure build settings.
+
+|ClusterVersion
+|Configure the desired version of the cluster and how updates are handled.
+
+|Console
+|Configure the web console.
+
 |DNS
-|See
+|Configure DNS.
+
+|Feature
+|Configure feature gates.
 
 |Image
-|See
+|Configure image registries.
 
 |Infrastructure
 |In {product-title} 4.0, you cannot customize the infrastructure resource.
 
 |Ingress
-|See
+|Configure Ingress.
 
 |Network
-|See
+|Configure the cluster networks.
 
 |OAuth
-|See
+|Configure the integrated OAuth server.
+
+|Project
+|Configure how projects are created.
```

Comment:
@sanchezl this one is also ours to doc, you should make sure the jira doc issue has subtasks for it

Reply:
Opened https://jira.coreos.com/browse/MSTR-356